DOC: legal process to perform a public release of a PyAnsys library (#236)
pyansys/dev-guide: doc/source/index.rst

.. include:: ../../README.rst

.. card:: Getting started

    .. grid:: 2
    
        .. grid-item-card:: :octicon:`list-unordered` What is PyAnsys?
           :link: getting-started/index
           :link-type: doc
    

>             A brief introduction to the PyAnsys Project and its ecosystem.

Reviewer: ```suggestion
            An introduction to the PyAnsys project and its ecosystem.
```
